lib/vdsm/storage/managedvolume.py

```diff
@@ -58,9 +58,14 @@
 DEV_RBD = "/dev/rbd"
 VOLUME_LINK_DIR = "/run/vdsm/managedvolumes/"
 
+# Drivers that have been tested and are known to work.
 SUPPORTED_DRIVERS = (
+    # Drivers tested by the oVirt team
     "rbd",
     "iscsi",
+
+    # Tested by Muli Ben-Yehuda
```

Comment:
feel free to also add a contact email address, say `<info (at) lightbitslabs.com>`